recipes/pydtmc/meta.yaml

```diff
@@ -0,0 +1,52 @@
+package:
+  name: pydtmc
+  version: "6.2.0"
+
+source:
+  url: https://pypi.io/packages/source/P/PyDTMC/PyDTMC-6.2.0.tar.gz
+  sha256: 4ea910105452b495721ebebd64163e5a33e42d82259e9b8d4c1c1fb6cfd4c293
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - python >=3.6
+    - pip
+    - setuptools
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - matplotlib
```

Comment:
Unless you actually need a PyQt gui, better to depend on `matplotlib-base`.

Reply:
Agreed, I'm gonna commit this change. Thanks!